src/solvers/smt2/smt2_tokenizer.h

```diff
@@ -56,6 +56,10 @@ class smt2_tokenizert:public parsert
     return messaget::error();
   }
 
+  /// skip any tokens until all parentheses are closed
+  /// or the end of file is reached
+  void skip_to_end_of_list();
```

Comment:
This is being added, but never invoked. (And I find it rather weird to have an a lexical analyzer.)

Reply:
Ok, now moved one level up to parser.